### ISSUE TITLE
Selenium.WebDriver 3.12.0 - .NET/C# Invalid Cast

### ISSUE THREAD

**Issue by ronjonesjr** [appium/appium#10694]
## The problem

It seems v 3.12.0 of the Selenium.WebDriver package has a bug in it that v 3.11.2 didn't have.  I have test code that locates buttons on a mobile application via AccessibilityId in Appium that works great in 3.11.2 but fails miserably in 3.12.0.  It's a Windows 10 application that is being tested via Appium's included WinAppDriver.

It looks like some type must have changed in the underlying system as the exception I get with 3.12.0 is:

System.InvalidCastException: Unable to cast OpenQA.Selenium.Remote.RemoteWebElement to IFindByAccessibilityId nor to IFindsByFluentSelector

The stack trace is:
MobileBy.FindElement(ISearchContext context)  <<-Exception is tossed here.
RemoteWebElement.FindElement(By by)

## Environment

* Appium Desktop 1.5.0 - I'd love to update but the latest desktop has bugs.
Windows 10

## Details
I reported this bug in NuGet for the Selenium.WebDriver package owners and I'm including this here as a reference.
https://www.nuget.org/packages/Selenium.WebDriver/

**Comment by mykola-mokhnach** [comment]
Please report to https://github.com/appium/appium-dotnet-driver
